<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>1021</width>
     <height>771</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="openTextButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addTextButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>50</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: lightgreen;
color: white;</string>
   </property>
   <property name="text">
    <string>Добавить текст</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
